mindsdb/utilities/config.py

```diff
@@ -43,7 +43,9 @@ def m1_0(config):
 
     def _validate(self):
         integrations = self._config.get('integrations', {})
-        for key in integrations:
+        for key, value in integrations.items():
+            if not isinstance(value, dict):
+                raise TypeError(f"Config error: integration '{key}' must be dict type")
```

Comment:
Maybe say "must be a json" instead of "must be dict type", since the users will be editing a JSON config.

Reply:
renamed